src/shared/Microsoft.Git.CredentialManager.Tests/StringExtensionsTests.cs

```diff
@@ -203,12 +203,12 @@ public void StringExtensions_TrimUntilIndexOf_String(string input, string trim,
         }
 
         [Theory]
-        [InlineData("fooTRIMbar", "TRIM", StringComparison.Ordinal, "bar")]
-        [InlineData("fooTRIMbar", "trim", StringComparison.Ordinal, "fooTRIMbar")]
-        [InlineData("fooTRIMbar", "tRiM", StringComparison.Ordinal, "fooTRIMbar")]
-        [InlineData("fooTRIMbar", "TRIM", StringComparison.OrdinalIgnoreCase, "bar")]
-        [InlineData("fooTRIMbar", "trim", StringComparison.OrdinalIgnoreCase, "bar")]
-        [InlineData("fooTRIMbar", "tRiM", StringComparison.OrdinalIgnoreCase, "bar")]
+        [InlineData("fooTRIMbarTRIMsoup", "TRIM", StringComparison.Ordinal, "barTRIMsoup")]
+        [InlineData("fooTRIMbarTRIMsoup", "trim", StringComparison.Ordinal, "fooTRIMbarTRIMsoup")]
+        [InlineData("fooTRIMbarTRIMsoup", "tRiM", StringComparison.Ordinal, "fooTRIMbarTRIMsoup")]
+        [InlineData("fooTRIMbarTRIMsoup", "TRIM", StringComparison.OrdinalIgnoreCase, "barTRIMsoup")]
+        [InlineData("fooTRIMbarTRIMsoup", "trim", StringComparison.OrdinalIgnoreCase, "barTRIMsoup")]
+        [InlineData("fooTRIMbarTRIMsoup", "tRiM", StringComparison.OrdinalIgnoreCase, "barTRIMsoup")]
```

Comment:
Maybe try "fooTrim" and make sure it yields "" for "Trim".

Didnt' see on first glance why did you update these?

Reply:
These are changes from a previous commit/PR that has been rebased away now.
The reason for the change was the _new_ command `TrimMiddle` cuts out the first 'trim' and leaves the prefix and suffix. This existing 'trim' method will cut out the prefix too, and I added a test to ensure it removes only the first match.